### PR TITLE
Update how-to-view-save-and-configure-build-log-files.md

### PR DESCRIPTION
**Pull request by yannduran**
Provide a method for people to save build output that's better and more convenient than copy/paste.

(You can replace all of this text with your description.)

Before creating your pull request, please check your content against these quality criteria:

- Did you consider search engine optimization (SEO) when you chose the title in the metadata section and the H1 heading (i.e. the displayed title that starts with a single #)?
- For new articles, did you add it to the table of contents?
- Did you update the "ms.date" metadata for new or significantly updated articles?
- Are technical terms and concepts introduced and explained, and are acronyms spelled out on first mention?
- Should this page be linked to from other pages or Microsoft web sites?

For more information about creating content for docs.microsoft.com, see the contributor guide at https://docs.microsoft.com/contribute/.

### CODE REVIEW

**Comment by PRMerger15** [comment]
@yannduran : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

File: docs/ide/how-to-view-save-and-configure-build-log-files.md [hunk 1]
```diff
@@ -12,7 +12,13 @@ ms.workload:
 ---
 # How to: View, save, and configure build log files
 
-After you build a project in the Visual Studio IDE, you can view information about that build in the **Output** window. By using this information, you can, for example, troubleshoot a build failure. For C++ projects, you can also view the same information in a *.txt* file that's created and saved automatically. For managed-code projects, you can copy and paste the information from the **Output** window into a *.txt* file and save it yourself. You can also use the IDE to specify what kinds of information you want to view about each build.
+After you build a project in the Visual Studio IDE, you can view information about that build in the **Output** window. By using this information, you can, for example, troubleshoot a build failure. 
+
+  - For C++ projects, you can also view the same information in a *.txt* file that's created and saved automatically. 
+
+  - For managed-code projects, you can click in the build output window, press **CTRL-S**, and Visual Studio will prompt you for a location to save the information from the **Output** window into a *.txt* file.
```
Comment (gewarren) [inline docs/ide/how-to-view-save-and-configure-build-log-files.md]:
```suggestion
  - For managed code projects, you can click in the build output window and press **Ctrl**+**S**. Visual Studio prompts you for a location to save the information from the **Output** window into a *.txt* file. 
```

File: docs/ide/how-to-view-save-and-configure-build-log-files.md [hunk 2]
```diff
@@ -24,9 +30,9 @@ If you build any kind of project by using MSBuild, you can create a *.txt* file
 
 1.  On the menu bar, choose **Build** > **Build Solution**.
 
-2.  In the **Output** window, highlight the information from the build, and then copy it to the **Clipboard**.
+2.  In the **Output** window, click somewhere in the text.
 
-3.  Open a text editor, such as **Notepad**, paste the information into the file, and then save it.
+3.  Press **CTRL-S** (Visual Studio will prompt you for a location to save the build output).
```
Comment (gewarren) [inline docs/ide/how-to-view-save-and-configure-build-log-files.md]:
```suggestion
3.  Press **Ctrl**+**S**.

   Visual Studio prompts you for a location to save the build output.
```

**Comment by PRMerger16** [comment]
@gewarren : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@gewarren : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

**Comment by yannduran** [comment]
@gewarren ahh, I'l ltry to remember that for the future

**Pull request merged**